source/optional_features/welsh_language/index.html.md.erb

```diff
@@ -7,9 +7,9 @@ weight: 132
 
 GOV.UK Pay supports Welsh-language payment pages.
 
-To use Welsh, you can include `"language": "cy"` in the body of a request to
-create a payment. Users will see Welsh text on the payment pages as they
-complete the [payment flow](/payment_flow).  If you do not specify a language,
+You can also [create payment links in Welsh](/payment_links).
```

Comment:
Not too sure this sentence fits here because the rest of the page is almost entirely about an API integration. Maybe this sentence should be right near the end, or possibly the next paragraph could have some exposition to make it really clear it’s only applicable to an API integration.

Reply:
Thanks @alexbishop1. As this more of a structure/style question, I'll make a note of this for the tech writer 2ier to consider.

Reply:
Our 2ier felt this was ok but recommended adding a brief intro to the payment links page (in https://github.com/alphagov/pay-tech-docs/pull/288/commits/1306a363b728c4ac9e3ef8d98254577747a11a2a), to help distinguish the two features.